### PR TITLE
policylibrary: add a catch all for setuid root and suid execution

### PR DESCRIPTION
**Pull request by tixxdz**

### CODE REVIEW

File: docs/content/en/docs/policy-library/observability/_index.md [hunk 1]
```diff
@@ -160,6 +161,39 @@ jq 'select(.process_exec != null) | select(.process_exec.process.binary | contai
...
+
+Monitor execution of the [setuid()](https://www.man7.org/linux/man-pages/man2/setuid.2.html) system calls family.
+
+### Use Case
+
+The [setuid()](https://www.man7.org/linux/man-pages/man2/setuid.2.html) and [setgid()](https://www.man7.org/linux/man-pages/man2/setgid.2.html)
+system calls family allow to change the effective user ID and group ID of the calling process.
+
+Detecting [setuid()](https://www.man7.org/linux/man-pages/man2/setuid.2.html) and [setgid()](https://www.man7.org/linux/man-pages/man2/setgid.2.html) calls that set the user ID or group ID to root is a common
+best-practice to identify when privileges are raised or still elevated.
```
Comment (kkourt) [inline docs/content/en/docs/policy-library/observability/_index.md:240]: What's the difference between raised or "still elevated"? Should we just keep raised?

Comment (tixxdz) [reply docs/content/en/docs/policy-library/observability/_index.md]: I kept raised for simplicity and let users can parse the events...

We also report case when an already effective uid 0 do setuid(0). For now it is fine, as we don't have the filters based on uids, but this is planned next to reduce noise anyway.

File: examples/policylibrary/privileges/privileges-setuid-root.yaml [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+#    - __ia32_sys_setfsuid
+#    - __x64_sys_setfsuid16
+#    - __ia32_sys_setfsuid16
+#
+# __sys_setfsgid
+#    - __x64_sys_setfsgid
+#    - __ia32_sys_setfsgid
+#    - __x64_sys_setfsgid16
+#    - __ia32_sys_setfsgid16
+#
```
[the remark this answers is not in the document]
Comment (olsajiri) [reply examples/policylibrary/privileges/privileges-setuid-root.yaml:51]: hm, but it's still separate syscall right? should we cover any possible way? I might misunderstood the purpose of that spec though

[the remark this answers is not in the document]
Comment (tixxdz) [reply examples/policylibrary/privileges/privileges-setuid-root.yaml:51]: Hi Jiri so the seteuid/setegid are wrappers https://www.man7.org/linux/man-pages/man2/seteuid.2.html check c library, internally they are calling the real system calls set**re**uid/set**re**gid  (there is no seteuid syscall). In this tracing library we handle all the syscalls around that, if you check, the setreuid/setregid are covered ;-)

Thank you Jiri for the review ;-)

**Review by olsajiri — APPROVED** [review]

**Review by kkourt — APPROVED** [review]

**Pull request merged**